tests/test_NoReservedWords.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+import pytest
+import xml.etree.ElementTree as ET
+import lsst.ts.xml as ts_xml
+
+
+def check_for_issues(csc, topic):
+	jira = ""
+	return jira
+
+
+@pytest.mark.parametrize("xmlfile,csc,topic", ts_xml.get_xmlfile_csc_topic())
+def test_no_reserved_words(xmlfile, csc, topic):
+	"""Test that the <EFDB_Name> field does not use any Reserved Words.
+
+	Parameters
+	----------
+	xmlfile : `pathlib.Path`
+		Full filepath to the Commands or Events XML file for the CSC.
+	csc : `test_utils.subsystems`
+		Name of the CSC
+	topic : `xmlfile.stem`
+		One of ['Commands','Events','Telemetry']
+	"""
+	saltype = "SAL" + topic.rstrip('s')
+	# Check for known issues.
+	jira = check_for_issues(csc, topic)
+	if jira:
+		pytest.skip(jira + ": " + str(xmlfile.name) + " <EFDB_Name> is not properly formed.")
+	# Test the <EFDB_Name> fields do not use Reserved Words.
+	with open(str(xmlfile), "r", encoding="utf-8") as f:
+		tree = ET.parse(f)
+	root = tree.getroot()
+	for name in root.findall(f"./{saltype}/item/EFDB_Name"):
+		assert name.text.upper() not in set(ts_xml.idl_reserved + ts_xml.db_reserved), \
+			'Reserved Word ' + name.text + ' used one or more times.'
```

Comment:
Please consider accumulating a list of violations, so the user see and fix all errors at once.

(What I had in mind is to use the loop to accumulate bad names in a list. Then after the loop assert that the list is empty.)

Reply:
Done.